lib/image_template.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+
+
+class ImageTemplate extends StatelessWidget {
+  String _assetName = 'assets/templates/LionSA1.svg';
+  double _dimension = 0.0;
```

Comment:
why do you need _dimension to be a class variable? It can be a variable in the build method